stories/core/Surface.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { CreeveyMeta } from 'creevey';
+import React from 'react';
+import { Surface, SurfaceProps } from '../../src/core';
+
+export default {
+  component: Surface,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    elevation: {
+      options: [2, 4, 8, 16, 24],
+    },
+  },
+  parameters: {
+    creevey: {
+      captureElement: null,
```

Comment:
Why added this?

Reply:
Without it, the test image was only capturing the border and not the box shadow around it

Reply:
Right.. Just now image captures whole iframe, which is also not needed.. But it is 1 image, so I guess its fine.